components/centraldashboard/public/components/activity-view.js

```diff
@@ -78,7 +78,7 @@ export class ActivityView extends PolymerElement {
             },
             activities: {
                 type: Array,
-                value: [],
+                value: () => [],
```

Comment:
Damn, I knew about this. I should have caught it in my review

Reply:
Ahh no biggie. I needed to initialize the array to something in order to use the splice and push methods but it never dawned on me that I'd be setting it up to use the same object for all instances of the element. It wouldn't have really mattered since we'd only ever have one `activity-view` on a page, but it made my tests fail when they ran in a particular order.